src/Block/GalleryListBlockService.php

```diff
@@ -173,8 +227,26 @@ public function configureSettings(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
+    public function getMetadata(): MetadataInterface
+    {
+        return new Metadata($this->getName(), null, null, 'SonataMediaBundle', [
+            'class' => 'fa fa-picture-o',
+        ]);
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     */
     public function getBlockMetadata($code = null)
     {
+        if ('sonata_deprecation_mute' !== (\func_get_args()[2] ?? null)) {
```

Comment:
Does this 2 need to be changed to 1?

Reply:
It depends on how are you providing the arguments for this method, but since there is no magic usage for argument 2 (offset 1 for `func_get_args()`), I think you must use it.